docs/shared-content-source/docs/modules/ROOT/partials/include.adoc

```diff
@@ -1,4 +1,34 @@
-// global attributes
-:page-partial:
 
-:cli-plugin: kubectl
+// global attributes if you update, make sure to update in docs/modules/RoOT:partials.adoc too
+:supported-java-v: >= 8
+:supported-sbt-v: version 1.2.8 or higher
+:supported-docker-v: version 18.09 or higher
+:supported-helm-v: version 3 or higher
+
+:kubernetes-platform-name:     Kubernetes
+:kubernetes-platform-version:  1.12
```

Comment:
this looks pretty outdated actually, I would omit the Kubernetes version since it will require continuous maintenance

Reply:
yes, agree.

Reply:
So, I removed kubernetes-platform-version.